Add check for forbidden keywords

JerrySpecter opened the pull request:
# Description

Add check for forbidden keywords. If the `projectName` argument gets a forbidden keyword it trows an error and prompts you to enter another name.

Example with CLI argument:
![image](https://user-images.githubusercontent.com/25869062/177146536-47befd51-af4b-4f40-a5f6-593357b626a4.png)

Example without arguments:
![image](https://user-images.githubusercontent.com/25869062/177146603-12d3bef2-7605-4505-85ba-96c35500341d.png)

iruzevic reviewed (approved):
looks good

iruzevic reviewed (changes requested):
we have tests in JS so please make this test cases

iruzevic commented on package.json:
```diff
@@ -64,6 +64,7 @@
 		"file-loader": "^6.2.0",
 		"husky": "^7.0.4",
 		"import-glob-loader": "^1.1.0",
+		"inquirer": "^9.0.0",
```
why is this package installed ? what is it used for?

iruzevic commented on package.json:
```diff
@@ -95,6 +96,7 @@
 		}
 	},
 	"devDependencies": {
+		"@babel/plugin-transform-modules-commonjs": "^7.18.6",
```
commnet from babel

JerrySpecter commented:
Pushed new changes, added tests, removed unused dependencies. Locally tests pass successfully, is there any way i can recreate Jest errors show here?
